### PR TITLE
[input] allow set of unicode in CAction

### PR DESCRIPTION
**Pull request by AlwinEsch**
## Description

This take also the unicode number of other CKey to construct.

This is required to get right character on addons in other countries.

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/input/Action.cpp [hunk 1]
```diff
@@ -75,7 +75,7 @@ CAction::CAction(int actionID, const std::string &name, const CKey &key):
     m_amount[i] = 0;
   m_repeat = key.GetRepeat();
   m_buttonCode = key.GetButtonCode();
-  m_unicode = 0;
+  m_unicode = key.GetUnicode();
```
Comment (AlwinEsch) [inline xbmc/input/Action.cpp]: @garbear was there a special reason why unicode was set to 0?

Comment (garbear) [reply xbmc/input/Action.cpp]: This code was c/p from the original code written 10 years ago. I suspect it's been wrong this whole time.

Comment (AlwinEsch) [reply xbmc/input/Action.cpp]: Then better bring this in? ;-)

Comment (garbear) [reply xbmc/input/Action.cpp]: Do it

**Pull request merged**